apps/meteor/app/models/server/raw/VideoConference.ts

```diff
@@ -145,4 +140,25 @@ export class VideoConferenceRaw extends BaseRaw<VideoConference> {
 			},
 		});
 	}
+
+	public async expireOldVideoConferences(minDate: Date): Promise<void> {
+		await this.update(
```

Comment:
`update` is deprecated, pls use `updateMany` or `updateOne`:
```suggestion
		await this.updateMany(
```